app/tests/unit/lib/theme/CustomDiffFields.test.tsx

```diff
@@ -127,73 +125,85 @@ describe("The Object Field Template", () => {
         formData={formData}
         formContext={{
           oldData: oldFormData,
+          latestCommittedData: oldFormData,
           oldUiSchema: oldUiTestSchema,
+          latestCommittedUiSchema: oldUiTestSchema,
           operation: "UPDATE",
         }}
       />
     );
 
     expect(componentUnderTest.getByText("stringTest NEW")).toBeInTheDocument();
     expect(componentUnderTest.getByText("stringTest OLD")).toBeInTheDocument();
+    expect(
+      componentUnderTest.queryByText(/^(.*?)LAST COMMITTED/)
+    ).not.toBeInTheDocument();
     expect(componentUnderTest.getByText("$100.00")).toBeInTheDocument();
     expect(componentUnderTest.getByText("$200.00")).toBeInTheDocument();
+    expect(componentUnderTest.queryByText("$300.00")).not.toBeInTheDocument();
     expect(
       componentUnderTest.getByText("I replaced the ID")
     ).toBeInTheDocument();
     expect(
       componentUnderTest.getByText("I replaced the OLD ID")
     ).toBeInTheDocument();
+    expect(
+      componentUnderTest.queryByText(/I am the last committed value/i)
+    ).not.toBeInTheDocument();
   });
-
-  it("shows data has been added when there is newData, the operation is 'CREATE' & there is no old data", () => {
+  // 101
+  it("shows data has been removed when there is old data and latest committed data but no new data", () => {
```

Comment:
this probably needs two cases, one for if old === latest and one for if it doesn't